compute_endpoint/globus_compute_endpoint/engines/helper.py

```diff
@@ -21,14 +24,16 @@
 log = logging.getLogger(__name__)
 
 _serde = ComputeSerializer()
+_RESULT_SIZE_LIMIT = 10 * 1024 * 1024  # 10 MiB
 
 
 def execute_task(
     task_id: uuid.UUID,
     task_body: bytes,
-    endpoint_id: t.Optional[uuid.UUID],
-    result_size_limit: int = 10 * 1024 * 1024,
-    run_dir: t.Optional[t.Union[str, os.PathLike]] = None,
+    endpoint_id: t.Optional[uuid.UUID] = None,
```

Comment:
You are importing future annotations, would be nice to remove `t.Optional` and `t.Union`